### DISCUSSION TITLE
How do we deploy Hooks with a current project or build with a current project?

### DISCUSSION THREAD

**Discussion by officialmohtechfirm** [RichardAH/rippled-hooks#67] [Q&A]
Greetings,

Thank you for this great step in bringing smart contract functionality to the XRPL. I am currently working on a project called RealXChange which plans on providing the market with many smart contracts solutions. My question is can we get a video tutorial or walkthrough like what is provided on the XRPL.org site on how to use hooks and test them exactly with a current project or with a new project. I understand the code behind it all but am wondering how I would actually take the deployed hook and use it in realtime testing/production. 

Thank you,
Mohammad

**Comment by RichardAH** [comment]
Hi. Thanks for your interest in Hooks. All information can be found here: https://hooks-testnet-v3.xrpl-labs.com/

**Reply by officialmohtechfirm** [reply]
Thank you
